pyproject.toml

```diff
@@ -14,16 +14,17 @@ documentation = "https://slowapi.readthedocs.io/en/latest/"
 include = ["slowapi/py.typed"]
 
 [tool.poetry.dependencies]
-python = "^3.6"
+python = "^3.6.2"
 limits = "^1.5"
 redis = "^3.4.1"
+anyio = "3.0.0"
 
 [tool.poetry.dev-dependencies]
 isort = "^4.3.21"
 mypy = "^0.761"
 black = "^19.10b0"
 fastapi = "^0.61.0"
-lxml = "^4.5.0"
+lxml = { version = "^4.5.0", markers="platform_system != 'Windows'" }
```

Comment:
I actually develop on Windows, which caused problems with this dependency. Whether or not it's included is kinda up to you, but I don't see any reason not to include the environment marker. Just means people on Windows can't build the documentation.

Reply:
I'm surprised it even works on windows, to be honest :)
But agree with you, I don't think there's an issue in adding the marker, I would think that most people will simply read the docs online rather than rebuild them locally.